build_tools/services.rb

```diff
@@ -158,9 +158,16 @@ def gem_dependencies(api, dependencies)
       core_version_string = "', '>= #{min_core}"
       dependencies['aws-sdk-core'] = "~> #{version_file.split('.')[0]}#{core_version_string}"
 
+      api['metadata'].fetch('auth', []).each do |auth|
+        if %w[aws.auth#sigv4 aws.auth#sigv4a].include?(auth)
+          dependencies['aws-sigv4'] = '~> 1.5'
```

Comment:
Curious to what significant happened with aws-sigv4 1.5 for this pessimistic version constraint?

Reply:
CRT for sigv4a.